app/models/beer.rb

```diff
@@ -5,17 +5,58 @@ class Beer < ActiveRecord::Base
   
 
   #method to list all users who have reviewed this beer?
-  def beer_reviewed_by
+  def user_reviews
+    list_of_reviews = self.reviews
+    list_of_reviews.map{|review| "User: #{review.user.name}, Rating: #{review.rating}, Review: #{review.description}"}
   end
 
+  
+
   #list of how many reviews a beer has
   def review_count
-
+    binding.pry
+    
   end
 
   #method to give highest rated beers
-  def highest_rated
+  def self.highest_rated
+    top_five_obj = Review.order(:rating).limit(5)
+
+    
+    puts "Here are our top 5 highest rated beers: #{top_five}"
+  end
+
+  #create methods to be able to sort by category(name, abv, rating)
+  def sort_beers_by_name
+    #list all beers by category, give the user the option for ASC or DESC
+    all_by_name = Beer.order(:name)
+    all_by_name
+  end
 
+  def sort_beers_by_abv
+    #list all beers by category, give the user the option for ASC or DESC
+    all_by_abv = Beer.order(:abv)
+    all_by_abv
+  end
+
+  def sort_beers_by_abv
+    #list all beers by category, give the user the option for ASC or DESC
+    all_by_rating = Review.order(:rating)
+    all_by_abv
```

Comment:
what's the all_by_abv method doing here? did you mean all_by_rating? also you dont need to return the variable twice `=` will automacally output the value